documentation/source/user_section/tutorials/lesion-analysis/lesion-segmentation/before-starting.rst

```diff
@@ -0,0 +1,20 @@
+Before starting this tutorial
+#############################
+
+#. Make sure that you have the following files in your working directory:
+
+   * ``single_subject/data/t2_lesion/t2_seg.nii.gz``: A 3D binary mask of the spinal cord.
+   * ``single_subject/data/t2_lesion/t2_lesion.nii.gz``: A 3D binary mask of the lesion.
+
+   You can get these files by downloading :sct_tutorial_data:`data_lesion.zip`. TODO: add data_lesion.zip
```

Comment:
A good point from @sandrinebedard: "search for msseg-challenge-2021, or maybe other challenges regarding MS subjects"

Reply:
The readme has info on where to find the dataset https://github.com/ivadomed/ms-challenge-2021/

Reply:
Thanks! I downloaded a bunch of T2w images from [here](https://www.kaggle.com/datasets/marwa96/isbi-ms-dataset), but all the images contain only the brain. The FOV is cropped right below the medulla.